### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by evilebottnawi** [mysticatea/eslint-plugin-node#35]
```json
{
    "rules": {
        "node/no-missing-require": [2, {
            "allowModules": ["jquery.cookie"]
        }]
    }
}
```
Output
```shell
Configuration for rule "node/no-missing-require" is invalid:
	Value "jquery.cookie" pattern mismatch.
```

**Comment by mysticatea** [comment]
Thank you for this issue.

I'll fix it today (JST).
